packages/components/CONTRIBUTING.md

```diff
@@ -6,20 +6,20 @@ The following is a set of guidelines for contributing to the `@wordpress/compone
 
 This set of guidelines should apply especially to newly introduced components. In fact, while these guidelines should also be retroactively applied to existing components, it is sometimes impossible to do so for legacy/compatibility reasons.
 
-For an example of a component that follows these requirements, take a look at [`ItemGroup`](/packages/components/src/item-group).
```

Comment:
Maybe we can add a more recent example? Or you prefer to have none?

Reply:
I think we have some many different implementations for components (naming conventions, monolithic vs compound, first party vs ariakit, sass vs CSS-in-JS, keeping as much logic in a hook vs in the render function...) that I'm hesitant to pick one as THE example. It also has potential to get out of date without us noticing.